packages/mc-scripts/src/utils/graphql-requests.ts

```diff
@@ -146,6 +172,24 @@ const fetchCustomApplication = async ({
   return customAppData.organizationExtensionForCustomApplication;
 };
 
+const fetchCustomView = async ({
+  mcApiUrl,
+  customViewId,
+}: TFetchCustomViewOptions) => {
+  const customViewData = await requestWithTokenRetry<
+    TFetchCustomViewFromCliQuery,
+    TFetchCustomViewFromCliQueryVariables
+  >(FetchCustomViewFromCli, {
+    variables: { customViewId },
+    mcApiUrl,
+    headers: {
+      'x-custom-view-id': customViewId,
+      'x-graphql-target': GRAPHQL_TARGETS.SETTINGS_SERVICE,
+    },
```

Comment:
We should we sending also the `x-application-id`

Reply:
I see, now it's passed in all query/mutation requests